frontend/src/metabase/collections/utils.ts

```diff
@@ -140,6 +144,7 @@ export function canPreviewItem(item: CollectionItem, collection: Collection) {
 export function canMoveItem(item: CollectionItem, collection: Collection) {
   return (
     collection.can_write &&
+    !isReadOnlyCollection(item) &&
```

Comment:
That's cheeky. So previously we were only checking to see if we could write to the collection the item was in, not the item itself? If so, this probably impacted other things as well.

Reply:
I don't think there are other situations where people have read but not write access to collections 🤔